### ISSUE TITLE
Mod folder BUG!!

### ISSUE THREAD

**Issue by NexusVick** [Nexus-Mods/Nexus-Mod-Manager#79]
I wanted to rename my mod folder but when I changed the path on nexus mod manager it still was looking for my old folder. relaunched NMM and made sure the path was correct and it was, so now when I launch skyrim it has no mods and the path is correct.

Launched NMM and it prompted me that the mod was missing and it wasn't. I had to literally make another mod folder (copy and paste) the entire game folder with the name that NMM wants. 

Now I have steam launching on one folder and NMM on the other folder, each folder is 70GIG big and they both got the same mods! Nexus mod manager path is listed that it installs mods on the game folder but I cant delete the cloned one with a different name and they are identical in content or NMM wont lauch if i delete it.

thank you for looking into it.

**Comment by Ghostwalker71** [comment]
It sounds like you changed the path to your game folder rather than your mod folder in NMM. You should be able to get it pointing back to the proper folder by rescanning for games.

**Comment by NexusVick** [comment]
Thank you so much for the reply.  I tried that already but just tried it again as you mentioned it to be extra sure and the mods and game are in the their proper folders within their proper perspective areas, at least that's what NMM says but that other folder that I named is tied in some how and when I scanned for games or check my directory that folder I made with a different name (the newly made folder) does not pop up, only the original mod folder and the original game folder. I cant go in my drive and delete that folder. I will need that folder as well with the original folder

So basically I have an extra 70GB I have to hold onto, sucks. thanks for the advice though.

Sent from my iPhone

On May 4, 2017, at 12:49 AM, Ghost <notifications@github.com<mailto:notifications@github.com>> wrote:


It sounds like you changed the path to your game folder rather than your mod folder in NMM. You should be able to get it pointing back to the proper folder by rescanning for games.

**Comment by Ghostwalker71** [comment]
In the settings menu there is a tab for the currently active game. Double check all the entries there and be sure they are correct.
The only other thing I can think of is that somehow the game ended up with a double entry, in which case the option I would choose is to run the game scan and deselect all games and finish, that should wipe all the path data from NMM, close and delete the duplicate folder, then reopen NMM and run the game scan again.
If that fails then I would uninstall NMM and delete the files that relate to NMM from the appdata folder as well. run a registry cleaner too to eliminate all broken registry links, then reinstall NMM. and run the game scan.

**Comment by NexusVick** [comment]
Thank you for the reply.  I tried all those tips and ended up with the same results. I honestly thought I was making an error on my side but I checked all paths several times and that looked like the paths are correct but it still lead me to another folder. I also tried clearing the path as you posted but ended up with the same results. At this point I think NMM got glitched on paths so I deleted NMM and renamed and placed the folders right where I want them then reinstalled NMM and that seems to be working fine now, but the fact that I had to delete the application just to correct the path is crazy. Besides this funky little glitch NMM has it's by far one of the greater Modding apps out there. :)

Thank you so much for the help you've been great!

Cheers!

Sent from my iPhone

On May 6, 2017, at 2:47 PM, Ghost <notifications@github.com<mailto:notifications@github.com>> wrote:


In the settings menu there is a tab for the currently active game. Double check all the entries there and be sure they are correct.
The only other thing I can think of is that somehow the game ended up with a double entry, in which case the option I would choose is to run the game scan and deselect all games so and finish, that should wipe all the path data from NMM, close and delete the duplicate folder, then reopen NMM and run the game scan again.
If that fails then I would uninstall NMM and delete the files that relate to NMM from the appdata folder as well. run a registry cleaner too to eliminate all broken registry links, then reinstall NMM. and run the game scan.
